pkg/openthread/include/ot.h

```diff
@@ -123,8 +126,9 @@ void openthread_bootstrap(void);
  * @param[in]  dev                pointer to a netdev interface
  * @param[in]  tb                 pointer to the TX buffer designed for OpenThread
  * @param[in]  rb                 pointer to the RX buffer designed for Open_Thread
+ * @param[in]  ab                 pointer to the ACK buffer designed for Open_Thread
```

Comment:
Shouldn't it be "designated" instead of "designed"? If so, would you mind to sneak in a fix of the doc of `tb` and `rb` (as separate commit) into this PR?

Reply:
Either one makes sense to me. So I can change as you want.